src/util.cpp

```diff
@@ -196,8 +196,15 @@ std::string AbsolutePath(const std::string &filepath) {
       char abs_path[MAX_PATH];
       return GetFullPathNameA(filepath.c_str(), MAX_PATH, abs_path, nullptr)
     #else
-      char abs_path[PATH_MAX];
-      return realpath(filepath.c_str(), abs_path)
+	  char* abs_path_temp = realpath(filepath.c_str(), nullptr);
```

Comment:
please format like the rest of the code: `*` on the right, `{` on same line, same indention, etc.